app/waf/src/controllers/schema.ts

```diff
@@ -246,9 +256,30 @@ function buildCollectionResponseSchema(
   };
 }
 
+export class TenantSchema {
+  static id = buildParameterSchema(
```

Comment:
27 // TODO: It is betther to have another decorator or something else
 28 // to specify the resource properties of request/response in model
 29 // definition, instead of re-define them here. Need to refactor
 30 // the code in the future.


As you mentioned, there is some known better way to do it, why not try it now. If we don't build the base now, it would become difficult to refactor in the future.

Reply:
This is no capacity to contain it in this sprint. Need to complete planned tasks at first.

Reply:
A 'good' design is not free, it required time to make the design. This pull request is to implement many-to-many relation, instead of refactor schema generating code. I can have a quick try with the existing decorate, in order to find out if we can make it a little better. If we cannot, let us consider that refactoring work in another thread, not in this PR. An open code review process is good, but it is not very good to freely stretch the scope during code review.

Reply:
Please look at #55